
%DIF LATEXDIFF DIFFERENCE FILE
%DIF DEL /tmp/benchmark.tex   Thu Feb 23 17:21:56 2023
%DIF ADD benchmark.tex        Thu Feb 23 17:20:39 2023
\documentclass[10pt,letterpaper]{article}
\usepackage[top=0.85in,left=0.85in,footskip=0.75in]{geometry}
% amsmath and amssymb packages, useful for mathematical formulas and symbols
\usepackage{amsmath,amssymb}
% Use adjustwidth environment to exceed column width (see example table in text)
\usepackage{changepage}
% Use Unicode characters when possible
\usepackage[utf8x]{inputenc}
% textcomp package and marvosym package for additional characters
\usepackage{textcomp,marvosym}
% cite package, to clean up citations in the main text. Do not remove.
%DIF 13c13
%DIF < \usepackage{cite}
%DIF -------
\usepackage[superscript]{cite} %DIF > 
%DIF -------
\usepackage{url}
% Use nameref to cite supporting information files (see Supporting Information section for more info)
%\usepackage{nameref,hyperref}
% line numbers
%\usepackage[right]{lineno}

%\usepackage{natbib}
\usepackage{graphicx}
\usepackage{float}
%DIF 23a23
\usepackage{authblk} %DIF > 
%DIF -------

% color can be used to apply background shading to table cells only
\usepackage[table]{xcolor}

% array package and thick rules for tables
\usepackage{array}
\usepackage{dsfont}

\usepackage{todonotes}
\usepackage{comment}

% referencing supplement-material
\usepackage{xr}
\externaldocument{./supplement}
%\usepackage{cleveref}



%% END MACROS SECTION

\input{./title.tex}
%DIF PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF UNDERLINE PREAMBLE %DIF PREAMBLE
\RequirePackage[normalem]{ulem} %DIF PREAMBLE
\RequirePackage{color}\definecolor{RED}{rgb}{1,0,0}\definecolor{BLUE}{rgb}{0,0,1} %DIF PREAMBLE
\providecommand{\DIFadd}[1]{{\protect\color{blue}\uwave{#1}}} %DIF PREAMBLE
\providecommand{\DIFdel}[1]{{\protect\color{red}\sout{#1}}}                      %DIF PREAMBLE
%DIF SAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddbegin}{} %DIF PREAMBLE
\providecommand{\DIFaddend}{} %DIF PREAMBLE
\providecommand{\DIFdelbegin}{} %DIF PREAMBLE
\providecommand{\DIFdelend}{} %DIF PREAMBLE
\providecommand{\DIFmodbegin}{} %DIF PREAMBLE
\providecommand{\DIFmodend}{} %DIF PREAMBLE
%DIF FLOATSAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddFL}[1]{\DIFadd{#1}} %DIF PREAMBLE
\providecommand{\DIFdelFL}[1]{\DIFdel{#1}} %DIF PREAMBLE
\providecommand{\DIFaddbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFaddendFL}{} %DIF PREAMBLE
\providecommand{\DIFdelbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFdelendFL}{} %DIF PREAMBLE
\newcommand{\DIFscaledelfig}{0.5}
%DIF HIGHLIGHTGRAPHICS PREAMBLE %DIF PREAMBLE
\RequirePackage{settobox} %DIF PREAMBLE
\RequirePackage{letltxmacro} %DIF PREAMBLE
\newsavebox{\DIFdelgraphicsbox} %DIF PREAMBLE
\newlength{\DIFdelgraphicswidth} %DIF PREAMBLE
\newlength{\DIFdelgraphicsheight} %DIF PREAMBLE
% store original definition of \includegraphics %DIF PREAMBLE
\LetLtxMacro{\DIFOincludegraphics}{\includegraphics} %DIF PREAMBLE
\newcommand{\DIFaddincludegraphics}[2][]{{\color{blue}\fbox{\DIFOincludegraphics[#1]{#2}}}} %DIF PREAMBLE
\newcommand{\DIFdelincludegraphics}[2][]{% %DIF PREAMBLE
\sbox{\DIFdelgraphicsbox}{\DIFOincludegraphics[#1]{#2}}% %DIF PREAMBLE
\settoboxwidth{\DIFdelgraphicswidth}{\DIFdelgraphicsbox} %DIF PREAMBLE
\settoboxtotalheight{\DIFdelgraphicsheight}{\DIFdelgraphicsbox} %DIF PREAMBLE
\scalebox{\DIFscaledelfig}{% %DIF PREAMBLE
\parbox[b]{\DIFdelgraphicswidth}{\usebox{\DIFdelgraphicsbox}\\[-\baselineskip] \rule{\DIFdelgraphicswidth}{0em}}\llap{\resizebox{\DIFdelgraphicswidth}{\DIFdelgraphicsheight}{% %DIF PREAMBLE
\setlength{\unitlength}{\DIFdelgraphicswidth}% %DIF PREAMBLE
\begin{picture}(1,1)% %DIF PREAMBLE
\thicklines\linethickness{2pt} %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\framebox(1,1){}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\line( 1,1){1}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,1){\line(1,-1){1}}}% %DIF PREAMBLE
\end{picture}% %DIF PREAMBLE
}\hspace*{3pt}}} %DIF PREAMBLE
} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbegin}{\DIFaddbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddend}{\DIFaddend} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbegin}{\DIFdelbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelend}{\DIFdelend} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbegin}{\DIFOaddbegin \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbegin}{\DIFOdelbegin \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbeginFL}{\DIFaddbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddendFL}{\DIFaddendFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbeginFL}{\DIFdelbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelendFL}{\DIFdelendFL} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbeginFL}{\DIFOaddbeginFL \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbeginFL}{\DIFOdelbeginFL \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
%DIF LISTINGS PREAMBLE %DIF PREAMBLE
\RequirePackage{listings} %DIF PREAMBLE
\RequirePackage{color} %DIF PREAMBLE
\lstdefinelanguage{DIFcode}{ %DIF PREAMBLE
%DIF DIFCODE_UNDERLINE %DIF PREAMBLE
  moredelim=[il][\color{red}\sout]{\%DIF\ <\ }, %DIF PREAMBLE
  moredelim=[il][\color{blue}\uwave]{\%DIF\ >\ } %DIF PREAMBLE
} %DIF PREAMBLE
\lstdefinestyle{DIFverbatimstyle}{ %DIF PREAMBLE
	language=DIFcode, %DIF PREAMBLE
	basicstyle=\ttfamily, %DIF PREAMBLE
	columns=fullflexible, %DIF PREAMBLE
	keepspaces=true %DIF PREAMBLE
} %DIF PREAMBLE
\lstnewenvironment{DIFverbatim}{\lstset{style=DIFverbatimstyle}}{} %DIF PREAMBLE
\lstnewenvironment{DIFverbatim*}{\lstset{style=DIFverbatimstyle,showspaces=true}}{} %DIF PREAMBLE
%DIF END PREAMBLE EXTENSION ADDED BY LATEXDIFF

\begin{document}
%\linenumbers
\maketitle

%Here I want to reference a figure that is in my supplementary content \cref{supp-fig:model_selection_criteria}.

%Here is a ref to the supplement, we have a Supplementary Figure \ref{fig:diff_vs_hela_find_a_better_label}.

\begin{abstract}

A frequent goal, or subgoal, when processing data from a quantitative shotgun proteomics experiment is a list of proteins that are differentially abundant under the examined experimental conditions. Unfortunately, obtaining such a list is a challenging process, as the mass spectrometer analyses the proteolytic peptides of a protein rather than the proteins themselves. We have previously designed a Bayesian hierarchical probabilistic model, Triqler, for combining peptide identification and quantification errors into probabilities of proteins being differentially abundant. \DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \DIFadd{However, the model was developed for data from data-dependent acquisition. }\DIFaddend Here we show that Triqler is \DIFaddbegin \DIFadd{also }\DIFaddend compatible with data-independent acquisition data after applying minor alterations for the missing value distribution. Furthermore, we find that it has better performance than \DIFdelbegin \DIFdel{other protein summarization tools when compared to }\DIFdelend a set of \DIFaddbegin \DIFadd{compared }\DIFaddend state-of-the-art \DIFdelbegin \DIFdel{DIA processing methods}\DIFdelend \DIFaddbegin \DIFadd{protein summarization tools when evaluated on data-independent acquisition data}\DIFaddend .
\end{abstract}

\section*{Introduction}

Mass spectrometry (MS)-based proteomics enables efficient detection of proteins in complex mixtures. There are several different techniques to make the technology quantitative \cite{bantscheff2007quantitative,kall2011computational}. Out of these, label-free quantification (LFQ) \cite{bondarenko2002identification} has the advantage that it can \DIFaddbegin \DIFadd{smoothly }\DIFaddend handle large sample sizes\DIFdelbegin \DIFdel{in a straightforward manner}\DIFdelend . For data from LFQ experiments, just as in other quantification schemes, there exists a plethora of processing options, all containing several processing steps, each subject to their different error sources, all affecting the result of the processing.


We have previously designed a hierarchical Bayesian model, Triqler, able to control for errors from both the identification and quantification process in LFQ experiments\DIFdelbegin %DIFDELCMD < \cite{The2018Integrated}%%%
\DIFdelend \DIFaddbegin \cite{the2019integrated}\DIFaddend . By integrating the error probabilities from identification and quantification, one can obtain better accuracy in calling differentially abundant proteins. Triqler \DIFdelbegin \DIFdel{was }\DIFdelend \DIFaddbegin \DIFadd{does so by assuming that peptide abundances follow a probability distribution. This abundance distribution is initiated according to a prior distribution, but updated based on each sample's registered peptide abundance values. This results in peptide abundance distributions that are integrated into protein abundance and subsequently fold change distributions. In the processing Triqler weights in information on differences in peptide abundances within sample groups and search engine identification error probabilities. Data that indicate uncertainty results in wider abundance distributions, while certainty results in tighter abundance distributions. Triqler also integrates the resulting fold change distributions to derive posterior probabilities for each protein having an actual fold change larger than a preset threshold value. These posterior probabilities are also averaged into $q$~values }\cite{the2019integrated}\DIFadd{.
}

\DIFadd{Triqler was originally }\DIFaddend designed for handling LFQ data from Data-dependent acquisition (DDA). However, \DIFdelbegin \DIFdel{many labs prefer }\DIFdelend \DIFaddbegin \DIFadd{in principle, once peptide abundances and identities are estimated, there are only small differences between data from DDA and }\DIFaddend Data-independent acquisition (DIA) mass spectrometry \cite{venable2004automated}\DIFdelbegin \DIFdel{as they find that it gives more reproducible peptide detection, and allows for a broader dynamical range in quantification }%DIFDELCMD < \cite{bern2010deconvolution,zhang2020DIA}%%%
\DIFdel{, compared to DDA. }%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \DIFadd{. }\DIFaddend Here, we set out to investigate Triqler's ability to summarize protein concentrations from peptide abundances derived from DIA data. We used the LFQBench from Navarro et al. \cite{navarro2016multicenter} for the evaluation. In the original benchmark, Navarro et al. included a comparison of different protein summarization strategies and found that the so-called Top3 method generally resulted in lower variance and better quantification accuracy than the built-in methods from OpenSwath, SWATH2.0, Skyline, Spectronaut, and DIA-Umpire\cite{navarro2016multicenter}. However, there are reasons to believe that more sophisticated methods would yield better protein quantification than the \DIFdelbegin \DIFdel{Top3-method}\DIFdelend \DIFaddbegin \DIFadd{Top3 method}\DIFaddend . Simple summarization methods based on mean and median peptide intensity have been shown to produce unreliable protein abundance estimates \cite{goeminne2015summarization}, and more advanced summarization strategies for LFQ data have been proposed in the literature \cite{silva2006absolute,cox2014accurate}. Summarization techniques such as PQPQ~\cite{forshed2011enhanced}, MSstats~\cite{choi2014msstats}, Diffacto~\cite{zhang2017covariation}, MSqRob2~\cite{sticker2020robust} and Triqler~\DIFdelbegin %DIFDELCMD < \cite{The2018Integrated} %%%
\DIFdelend \DIFaddbegin \cite{the2019integrated} \DIFaddend have all been shown to outperform Top3 and there are no theoretical reasons why the stated methods would not perform well for DIA data. Hence, we found it apt to \DIFdelbegin \DIFdel{benchmark Triqler }\DIFdelend \DIFaddbegin \DIFadd{modify Triqler to handle DIA data, and to benchmark it }\DIFaddend against a set of state-of-the-art protein summarization methods\DIFdelbegin \DIFdel{using peptide quantities from the LFQBench DIA data set}\DIFdelend .

\section*{Materials and methods}

\DIFdelbegin \subsection*{\DIFdel{Data description}}
%DIFAUXCMD
\DIFdelend \subsubsection*{LFQBench mass spectrometry data}

We downloaded the LFQBench dataset~\cite{navarro2016multicenter} from \DIFdelbegin \DIFdel{PRIDE }\DIFdelend \DIFaddbegin \DIFadd{ProteomeXchange }\DIFaddend (PXD002952). Here we used the TripleTOF6600 section of the study, which was harvested with a setup of 32 fixed windows MS2-windows. We also restricted ourselves to the low ratio difference samples, referred to as the HYE124 hybrid proteome samples in the original study. These consist of triplicates of Sample A composed of tryptically digested proteins from 65\% w/w HeLa, 30\% w/w yeast, and 5\% w/w \textit{E. coli} cells, and triplicates of Sample B, composed of 65\% w/w, 15\% w/w yeast, and 20\% w/w \textit{E. coli} proteins. Samples from HYE110 and the TripleTOF5600 section of PXD002952 were omitted in this study. Further details about mass spectrometric instrumentation and data acquisition are available in Navarro et al.~\cite{navarro2016multicenter}. The \verb|.wiff| files were converted to \verb|.mzML| files in a centroided format using msconvert (using Windows OS msconvert version 3.0) with peakPicking filter msLevel=1-). 

\subsubsection*{LFQBench sequence database}

Uniprot FASTA files with one protein sequence per gene were downloaded for each species (UP000005640, UP000000625, and UP000002311, acquired on 2021-06-16). The unfiltered FASTA files contained 20 590 human proteins, 6 046 yeast proteins, and 4 373 \textit{E. coli} proteins. To reduce the effect of the different protein inference strategies for the tested protein summarization tools, a modified FASTA file, without shared peptides, was used for database search. The filter removed protein sequences with shared peptides so that the final database did not contain any two proteins sharing tryptic peptides longer than 7 amino acids. After filtering the FASTA file contained 20 302 proteins (288 human proteins fewer proteins than \DIFaddbegin \DIFadd{in }\DIFaddend an unfiltered database), 5 848 yeast proteins (198 yeast proteins fewer proteins than \DIFaddbegin \DIFadd{in }\DIFaddend an unfiltered database), and 4 306 \textit{E. Coli} proteins (67 \textit{E. Coli} proteins than \DIFaddbegin \DIFadd{in }\DIFaddend an unfiltered database). Replacing the I/L amino acids to handle the mass equivalence did not result in any considerable differences (See Supplementary Table \ref{table:proteins_in_database}). We also added pseudo-reverse sequences to the database as decoys for target-decoy analysis using OpenSwathDecoyGenerator. 

\DIFaddbegin \subsubsection*{\DIFadd{Cancer proteomics data}}

\DIFadd{We also downloaded two DIA data cancer proteomics-related datasets from ProteomeXchange, with accession numbers PXD004684 and PXD022992.  The first set consists of data from four frozen lung squamous cell carcinomas and four adjacent tissues harvested with a Thermo QExactive Plus~}\cite{stewart2017relative}\DIFadd{. The second set consists of data from a comparison of three primary cell lines (SK-MEL-1, A375, and G-361) to three metastatic (RPMI-7951, SH-4, and SK-MEL-3) cell lines analyzed (two samples per cell line) by a Thermo Orbitrap Fusion Lumos Tribrid~}\cite{gao2021data}\DIFadd{. 
}

\DIFaddend \subsection*{General workflow}

We used \DIFdelbegin \DIFdel{two }\DIFdelend \DIFaddbegin \DIFadd{three }\DIFaddend separate strategies to generate peptide abundances from the DIA runs. First, we used a spectral library consisting of selected spectra from separate DDA runs, we refer to this workflow as ID\DIFdelbegin \DIFdel{hereon, and }\DIFdelend \DIFaddbegin \DIFadd{, }\DIFaddend secondly, we searched pseudo-spectra generated directly from the DIA data, we will refer to this workflow as PS\DIFdelbegin \DIFdel{from hereon. The workflows are shown in Figure \ref{fig:flowchart}}\DIFdelend \DIFaddbegin \DIFadd{, and thirdly we applied a DIA-NN centered pipeline that also operated directly on DIA data}\DIFaddend .  We will describe the parameter choices of \DIFdelbegin \DIFdel{both these two }\DIFdelend \DIFaddbegin \DIFadd{these }\DIFaddend methods below.


\DIFdelbegin %DIFDELCMD < \begin{figure}[htp]
%DIFDELCMD <     \centering
%DIFDELCMD <     \includegraphics[width=1.0\linewidth]{./img/methods.png} 
%DIFDELCMD <     %%%
%DIFDELCMD < \caption{%
{%DIFAUXCMD
%DIFDELCMD < {\bf %%%
\DIFdelFL{The DDA identification-driven matching (ID) and pseudo-spectrum workflow pipelines (PS).}%DIFDELCMD < } %%%
\DIFdelFL{In the pipelines DIA-Umpire(SE), MSFragger and EasyPQP are run from Fragpipe (}%DIFDELCMD < \protect\url{https://fragpipe.nesvilab.org/}%%%
\DIFdelFL{). A 1\% PSM-level identification threshold was applied to Top3 and MSstats, a 1\% peptide-level threshold to MSqRob2, while all the identifications were provided unfiltered to Triqler. }%DIFDELCMD < \label{fig:flowchart}%%%
}      
%DIFAUXCMD
%DIFDELCMD < \end{figure}
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \subsubsection*{DDA identification-driven (ID) spectral library}

For the ID workflow, we searched the \DIFdelbegin \DIFdel{LFQBench provided }\DIFdelend \DIFaddbegin \DIFadd{LFQBench-provided }\DIFaddend DDA runs with MSFragger\cite{kong2017msfragger} with default settings\DIFdelbegin \DIFdel{(Precursor mass tolerance of }%DIFDELCMD < [%%%
\DIFdel{-20, 20}%DIFDELCMD < ] %%%
\DIFdel{ppm, fragment tolerance of 20 ppm, Calibration and Optimization: ``Max calibration, parameter optimization'', isotope error: 0/1, Data type: DDA, Load rules: stricttrypsin, Cut after: KR, Cleavages: ENZYMATIC, Missed cleavages: 2, Clip N-term M: True, Peptide length of }%DIFDELCMD < [%%%
\DIFdel{7, 50}%DIFDELCMD < ]%%%
\DIFdel{, Peptide mass range of }%DIFDELCMD < [%%%
\DIFdel{500, 5000}%DIFDELCMD < ]%%%
\DIFdel{, Split database: 1, and allowing for oxidation on methionine and protein N-terminus modifier as variable modifications)}\DIFdelend , and a spectral library was constructed with EasyPQP \cite{easypqp} with the default setting\DIFdelbegin \DIFdel{(RT Calibration: ``Automatic selection of a run as reference RT'', RT Lowess Fraction: 0.1, UniMod annotation tol(Da), Fragment annotation tol(ppm): 15, and the default PSM-level threshold of 0.01, peptide-level false discovery rate (FDR) of 0.01, and protein-level FDR of 0.01)}\DIFdelend \DIFaddbegin \DIFadd{. No transition refinement was used}\DIFaddend . OpenSwatchDecoyGenerator was used to generate decoys for the spectral library with a pseudo-reverse method. The DIA data \DIFdelbegin \DIFdel{was }\DIFdelend \DIFaddbegin \DIFadd{were }\DIFaddend searched with the spectral library through OpenSwath Workflow, and the \texttt{m\_score} was computed using PyProphet\cite{teleman2015diana}. The \texttt{m\_score} cut-off for a user-specified peptide identification \DIFdelbegin \DIFdel{false-discovery }\DIFdelend \DIFaddbegin \DIFadd{false discovery }\DIFaddend rate was computed with SWATH2STATS \cite{blattmann2016swath2stats}. This process resulted in a set of detected peptides together with their assessed peptide identification accuracies and abundance estimates (Supplementary Table \ref{fig:osw_peptide_and_protein_id} shows the number of identified peptides and proteins).


\subsubsection*{Pseudo-spectra generation (PS) spectral library}

For the PS workflow, we also used the FragPipe software, employing DIA-Umpire to extract pseudo-spectra from the DIA data. The DIA-Umpire parameters \DIFdelbegin \DIFdel{was }\DIFdelend \DIFaddbegin \DIFadd{were }\DIFaddend set to default values\DIFdelbegin \DIFdel{(MS1 PPM: 10, MS2 PPM: 20, Max Missed Scans: 1, Mass Defect Filter: True, RP max: 25, RF max: 500, Corr Threshold: 0, Delta Apex: 0.2, RT Overlap 0.3, Mass Defect Offset 0.1, Isotope Pattern: 0.3, MS1 SN: 1.1, MS2 SN: 1.1, Adjust Fragment Intensity: True)}\DIFdelend . The pseudo-spectra were subsequently searched using MSFragger with default settings\DIFdelbegin \DIFdel{(Precursor mass tolerance of }%DIFDELCMD < [%%%
\DIFdel{-20, 20}%DIFDELCMD < ] %%%
\DIFdel{ppms, fragment tolerance of 20 ppms, Calibration and Optimization: ``Max calibration, parameter optimization'', isotope error: 0/1, Data type: DDA, Load rules: stricttrypsin, Cut after: KR, Cleavages: ENZYMATIC, Missed cleavages: 2, Clip N-term M: True, Peptide length of }%DIFDELCMD < [%%%
\DIFdel{7, 50}%DIFDELCMD < ]%%%
\DIFdel{, Peptide mass range of }%DIFDELCMD < [%%%
\DIFdel{500, 5000}%DIFDELCMD < ]%%%
\DIFdel{, Split database: 1, and allowing for oxidation on methionine and protein N-terminus modifier as variable modifications)}\DIFdelend . A spectral library was built from the resulting PSMs using easyPQP with default \DIFdelbegin \DIFdel{setting (RT Calibration: ``Automatic selection of a run as reference RT'', RT Lowess Fraction: 0.1, UniMod annotation tol(Da), Fragment annotation tol(ppm): 15, and the default PSM-level threshold of 0.01, peptide-level FDR of 0.01, and protein-level FDR of 0.01)}\DIFdelend \DIFaddbegin \DIFadd{settings}\DIFaddend . DIA-NN\DIFaddbegin \DIFadd{~}\cite{demichev2020dia} \DIFadd{(version 1.7.12) }\DIFaddend was used for peptide quantifications with \DIFdelbegin \DIFdel{settings specified in }%DIFDELCMD < \url{https://fragpipe.nesvilab.org/docs/tutorial_DIA.html#quantify-with-dia-nn} %%%
\DIFdel{(Protein inference: ``Off'', Quantification strategy: ``Robust LC (High accuracy)'', Precursor FDR (\%): 1.0, Protease: ``Trypsin/P'', Missed cleavages: 1, Maximum number of variable modifications: 0, N-term M excision: True, C carbamidomethylation: True, Peptide length: }%DIFDELCMD < [%%%
\DIFdel{7, 30}%DIFDELCMD < ]%%%
\DIFdel{, Precursor m/z range: }%DIFDELCMD < [%%%
\DIFdel{300, 1800}%DIFDELCMD < ]%%%
\DIFdel{, Fragment ion m/z range: }%DIFDELCMD < [%%%
\DIFdel{200, 1800}%DIFDELCMD < ]%%%
\DIFdel{, Mass accuracy: 0.0}\DIFdelend \DIFaddbegin \DIFadd{default settings. 
}

\subsubsection*{\DIFadd{Directly extracted peptide abundances}}

\DIFadd{For this workflow, we used the quantms nf-core pipeline (}\url{https://nf-co.re/quantms}\DIFadd{)}\DIFaddend , \DIFdelbegin \DIFdel{MS1 accuracy: 0.0, Scan window: 0, Use isotopologues: True, Remove likely interferences: True, Neural network classifier: ``Single-pass mode'', Protein inference: Genes, Cross-run normalization: ``RT-dependent'', Library generation: ``Smart profiling''). To compute false discovery rates, }\DIFdelend \DIFaddbegin \DIFadd{which converts raw files to mzML with thermorawfileparser~}\cite{hulstaert2019thermorawfileparser} \DIFadd{and subsequently extracts peptide abundances using }\DIFaddend DIA-NN\DIFdelbegin \DIFdel{uses a built-in custom implementation of the mProphet algorithm}%DIFDELCMD < \cite{reiter2011mprophet, demichev2020dia} %%%
\DIFdel{(Supplementary Table \ref{fig:diann_peptide_and_protein_id} shows the number of identified peptides and proteins)}\DIFdelend \DIFaddbegin \DIFadd{~}\cite{hulstaert2019thermorawfileparser}\DIFadd{. The pipeline generated decoys for FDR calculations which were discarded after DIA-NN processing. To circumvent the lack of decoys in output for Triqler we concatenated shuffled entrapment sequences in the FASTA database }\cite{granholm2013determining}\DIFadd{. This workflow was used with the two cancer proteomics-related datasets}\DIFaddend .

\subsection*{Protein summarization}

The peptide quantities were summarized to proteins using the average of the three most intense peptides (We call it Top3), MSstats, MSqRob2, and Triqler. This is done for \DIFdelbegin \DIFdel{both the ID and PS }\DIFdelend \DIFaddbegin \DIFadd{all three }\DIFaddend pipelines. 

\subsubsection*{Top3}

We implemented a short script that takes the average of the 3 most abundant PSMs for each protein and sample. In samples only having two PSMs these were also included, still represented by their average. Proteins with one or zero PSMs per sample were excluded. PSMs were filtered at a 1\% PSM-level FDR before performing the Top3 protein summarization. 

\subsubsection*{MSstats}

We installed MSstats version 3.18.5 using R/Bioconductor (available at \url{https://www.bioconductor.org/packages/release/bioc/html/MSstats.html}). MSstats use feature-level data, allowing for multiple PSM hits per peptide identification. We \DIFaddbegin \DIFadd{use the disaggregate function to disaggregate the PSMs to fragment-level data. We }\DIFaddend filtered so that every protein had at least 2 peptides and a maximum of 10 peptides, and we thresholded with a \texttt{m\_score} which corresponds to a peptide identification FDR lower than 0.01 for both ID and PS pipelines. \DIFaddbegin \DIFadd{For the cancer data set we allowed up a maximum of 100 peptides. }\DIFaddend In the PS pipeline \DIFaddbegin \DIFadd{and nf-core pipeline}\DIFaddend , the data was filtered on the \texttt{Q.Value} columns from the DIA-NN output file. In the ID pipeline, we computed an \texttt{m\_score} corresponding to a 1\% FDR and used this \texttt{m\_score} to filter the data. MSstats was run using the MSstats command \texttt{dataProcess}. The significance testing between conditions was performed using the MSstats function \texttt{groupComparison}.  

\subsubsection*{MSqRob2}
%DIF < 
\DIFaddbegin 

\DIFaddend We installed MSqRob2 version 0.9 using R/Bioconductor (available at \url{https://github.com/statOmics/msqrob2}). MSqRob2 takes peptide-level input. The output from OpenSwath and DIA-NN is at PSM-level. We select the top PSM hit as our peptide and filter the data on 1\% peptide level FDR to get peptide-level data. The \DIFdelbegin \DIFdel{highest scoring }\DIFdelend \DIFaddbegin \DIFadd{highest-scoring }\DIFaddend PSMs are selected by the highest \texttt{m\_score} for OpenSwath and highest \texttt{CScore} for DIA-NN. MsqRob2 was run using the MSqRob2 command \texttt{msqrob}, where the contrast was set to \texttt{condition}.
MSqRob2 uses \texttt{lme4} to construct a linear-mixed model with random effect, but without fixed effect. 
\DIFdelbegin \DIFdel{The models specified by the variable formulas construct the models $y = Z_1 \mu + \epsilon$ and $y = Z_2 \mu + \epsilon$, where $Z_1$ contains the random effects condition, sample and feature (peptides for protein) and $Z_2$ contains only a random effect for the condition. Some proteins have only intensities from one peptide. This can cause the first model $y = Z_1 \mu + \epsilon$ parameters to fail to converge. For these proteins, we use the reduced model $y = Z_2 \mu + \epsilon$.  
%DIF < clear
}\DIFdelend \DIFaddbegin 

\subsubsection*{\DIFadd{Limma analysis on DIA-NN protein groups}}

\DIFadd{We downloaded limma}\cite{limma} \DIFadd{version 3.50.3 from Bioconductor. We applied limma with default parameters to the protein groups inferred from DIA-NN in the PS pipeline.  
}


\DIFaddend \subsubsection*{Triqler}

We downloaded Triqler from \url{https://github.com/statisticalbiotechnology/triqler}. We used Triqler v0.6.1 for the tests described in this paper. We selected a lower bound estimate for the \texttt{fold\_change\_eval} parameter as described in The\&K\"{a}ll \cite{the2021triqler}, which ended up as 0.76 for the spectral library data, and 0.51 for the pseudo-spectra enabled data.

As Triqler's model accounts for the assessed uncertainty in the prior identification steps of the data, it does not threshold the PSMs but instead takes all of the PSMs as input. The \texttt{searchScore} column should reflect increasing certainty in PSMs. Therefore we apply the negative log-transform to the \texttt{m\_score} or \texttt{Q.Value} to indicate \texttt{searchScore} for the two different pipelines. 

\DIFaddbegin \DIFadd{When evaluating Triqler we estimated a fold-change valuation thresholds using a previously described heuristic that ensures that $99\%$ of the 
probability distribution of the normal distribution is contained~}\cite{the2021triqler}\DIFadd{, as,
}\begin{equation}
 \DIFadd{L = \frac{\log_2 \left( 10^{\sigma_y} \right) \cdot \sqrt{2}}{\sqrt{N}} \cdot 
2.5,
}\end{equation}
\DIFadd{where $\sigma_y$ is the standard deviation of the protein prior in $\log_{10}$ 
abundance units, $N$ is the number of samples in the group. Triqler estimates and prints this minimum advisable fold change threshold by default, and it can be retrieved from Triqlers standard output.
Also, Triqler removes any peptides that are attributed to more than one protein.
}

\DIFaddend \subsubsection*{Multiple Hypothesis Correction}
\DIFaddbegin 

\DIFaddend There was a slight difference in some of the benchmarking metrics, as the multiple test correction is performed with $q$~value for Triqler and Top3, while MsStats and MSqRob2 use Benjamini-Hochberger \cite{benjamini1995controlling} corrections. The $q$~value approach aims to give an unbiased estimator of FDR, while the Benjamini-Hochberger approach estimates the upper bound of the FDR (and will therefore result in an FDR equal to or higher than the $q$~value). As a consequence, the statistics from MSstats and MSqRob2 should be more conservative than the estimates from Triqler and Top3\cite{korthauer2019practical}.

\DIFdelbegin %DIFDELCMD < \begin{comment}%DIFDELCMD < 
%DIFDELCMD < \subsection*{Application to non-engineered data}
%DIFDELCMD < To show that our results are relevant for real application we reanalyze the OXA-dataset. We group the data as in Yang et al. \cite{YANG2022104682}. The proteins in cluster 1(C1) and 2(C2) were continually up-regulated or down-regulated from Ctrl to ST and from ST to LT. The proteins in cluster 3(C3) and 4(C4) were proteins up-regulated or down-regulated from ST to LT, and the proteins in cluster 5(C5) and 6(C6) were up-regulated or down-regulated from Ctrl to ST.  
%DIFDELCMD < \end{comment}
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \section*{Results}

To establish that Triqler is useful when evaluating DIA data, we first examined the DIA data to establish that the abundance values derived from such processes are in line with the assumptions that Triqler makes about input data. Second, we benchmarked Triqler against a set of state-of-the-art methods for protein summarization. For both tasks, we used selected parts of the LFQBench dataset, which we processed in two different ways. We used a pipeline with a DDA identification-driven (ID) spectral library, as well as one using a Pseudo-spectra generation (PS) spectral library (See Methods).


\subsubsection*{Validations of properties of DIA peptide abundance}

DIA data is assumed to encompass a larger dynamic range than DDA data \citation{bilbao2015processing}. This could affect one of Triqler's assumptions, that the noise structure is mainly multiplicative, i.e. that the standard deviation within a sample group is proportional to its mean. When investigating all the peptide abundance measurements at a 1\% PSM-level identification false discovery rate (FDR) from the TripleTOF6600 section of the LFQBench dataset, we found a relatively linear relationship between standard deviation and mean (Supplementary Figure \ref{fig:uniform_offset_in_standard_deviation_boxplot}A-D). 

However, when \DIFdelbegin \DIFdel{Triqler tried to estimate }\DIFdelend \DIFaddbegin \DIFadd{estimating }\DIFaddend the distribution of missing values \DIFaddbegin \DIFadd{with Triqler}\DIFaddend , we noted that the lower number of missing values in DIA compared to DDA (\DIFdelbegin \DIFdel{<2\% }\DIFdelend \DIFaddbegin \DIFadd{$<2\%$ }\DIFaddend for DIA), \DIFdelbegin \DIFdel{lead }\DIFdelend \DIFaddbegin \DIFadd{led }\DIFaddend to parameter estimates out of the expected range. We hence introduced an alternative estimation procedure for DIA data\DIFaddbegin \DIFadd{. Instead of fitting a censored normal distribution function over all XIC values as for DDA data, we use the mean value of the missing values to fit a censored normal distribution function }\DIFaddend (See Supplementary Figure \ref{fig:fraction_missing_values})\DIFdelbegin \DIFdel{that can be }\DIFdelend \DIFaddbegin \DIFadd{. The alternative curve fitting procedure is }\DIFaddend enabled by a command line option in Triqler \DIFdelbegin \DIFdel{. 
}\DIFdelend \DIFaddbegin \DIFadd{named }{\color{blue}%DIFAUXCMD
\verb|--missingValuePrior=DIA|
%
}%DIFAUXCMD
\DIFaddend 

\subsubsection*{Harmonization of protein inference procedures.}

Encouraged by the finding that peptide abundances from DIA data have similar properties to those of DDA data, we wanted to see how well Triqler works in practice. Particularly, we wanted to compare the performance of Triqler to that of other protein summarization methods. However, to do so we first needed to establish some principles for how to benchmark summarization methods on data from whole-cell mixtures, when comparing methods that use different protein inference procedures \cite{serang2012recognizing}. When reporting the number of differentially abundant proteins in data sets from mixtures from whole-cell extracts, a protein inference scheme that infers any protein containing a detected peptide will report more differentially abundant proteins than a more restrictive scheme that just reports a parsimonious set of proteins. There are no restrictive mechanisms detecting situations where non-present proteoforms are reported as long as they are reported with protein abundance rates compatible with the right proteome. To alleviate, or at least minimize, this problem from our comparison, we restricted the searched sequence database by removing proteins with shared peptides in such a manner that no two proteins shared a tryptic peptide longer than 7 amino acids. This operation aims at a fairer comparison of protein summarization regardless of the protein inference method. 

\subsubsection*{Distributions of the estimated fold changes}

We applied Triqler, MSstats, MSqRob2, and Top3 (see Methods) to the peptide abundances derived from the ID and PS workflows from the varying concentrations of {\em E. Coli} and yeast concentrations in a background of HeLa-cells of the LFQBench set. Although Triqler is not meant as a tool to give point estimates, we here used Triqler's maximum a \DIFdelbegin \DIFdel{posterioris }\DIFdelend \DIFaddbegin \DIFadd{posteriori probability }\DIFaddend (MAPs) as point estimates of protein abundance to enable comparisons to other methods. Also, for the comparisons, we removed the methods' fold change selection mechanisms. For an overview of the results, we made histograms of the methods' estimated protein-level fold changes (Figure \ref{fig:fc_histogram} and Supplementary Figure \ref{fig:fc_histogram_supplement}), 

Triqler and Top3 \DIFdelbegin \DIFdel{appears }\DIFdelend \DIFaddbegin \DIFadd{appear }\DIFaddend to have less fold change bias than MSstats and MSqRob2, that is, the apex of the distributions is centered more closely to the lysate mixture rates. MSstats and MSqRob2 have distribution apexes that, for each species, are centered at lower fold changes than the true lysate mixture rates. Reprocessing the data with replaced labels led to the reversed result, i.e. the apexes centered around values higher than the pipetted mixture rates for both MSstats and MSqRob2 (data not shown). We also observe that the apex of the distributions has higher values for Triqler than the other methods in Figure \ref{fig:fc_histogram}. We provide comparisons of the fold change estimates in Figure \DIFdelbegin \DIFdel{\ref{fig:scatter_triqler_vs_method} }\DIFdelend \DIFaddbegin \DIFadd{\ref{fig:fc_scatter} }\DIFaddend and Supplementary Figure \ref{fig:fc_scatter_triqler_vs_method_supplement}.


\DIFdelbegin \DIFdel{Triqler has }\DIFdelend \DIFaddbegin \DIFadd{As visible in Figure \ref{fig:fc_histogram}, Triqler's fold change distributions have }\DIFaddend longer tails for the estimated yeast and {\em E. Coli} \DIFdelbegin \DIFdel{fold change distributions}\DIFdelend \DIFaddbegin \DIFadd{than the compared methods}\DIFaddend . These values are explained by proteins quantified with \DIFdelbegin \DIFdel{fewer peptides, i.e. less certain protein quantification}\DIFdelend \DIFaddbegin \DIFadd{less certainty}\DIFaddend . Proteins with correctly estimated fold changes are identified by more peptides than those differing significantly from their expected fold change (data not shown). \DIFaddbegin \DIFadd{The technical aspect of this is explained by how Triqler computes the posteriors for differentially abundant proteins; with better data, more confidence is put on data and less confidence is put on the prior distributions, resulting in narrower fold change distributions with means more centered at the observed mean values. 
}\DIFaddend 


\begin{figure}[hbt]
    \centering
    \begin{tabular}{cc}
	    \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/histogram_ID_triqler.png} %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.4\linewidth]{./img/histogram_ID_triqler.png} \DIFaddendFL & 
	    \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/histogram_ID_top3.png} %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.4\linewidth]{./img/histogram_ID_top3.png} \DIFaddendFL \\ 
        A & B \\ 
	    \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/histogram_ID_msstats.png} %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.4\linewidth]{./img/histogram_ID_msstats.png} \DIFaddendFL & 
	    \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/histogram_ID_msqrob2.png} %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.4\linewidth]{./img/histogram_ID_msqrob2.png} \DIFaddendFL \\
        C & D 
    \end{tabular}
    \caption{{\bf Comparison of reported fold change distributions.} The summarized protein abundances from the ID spectral library pipeline for (A) Triqler, (B) Top3, (C) MSstats, and (D) MSqRob2 were binned and plotted as histograms. The dashed lines indicate the pipetted log2-fold change ratio between a specie and HeLa samples. We noted that the empirical densities of the protein counts are less biased for Triqler and Top3 than MSstats and MSqRob2, as the apex of their distribution was found closer to the true fold change difference (see Supplementary Figure \ref{fig:fc_histogram_supplement} for reported fold change distributions for both ID and PS workflows). For these histograms, we included all the summarized proteins and did not remove any proteins based on  the significance or fold change thresholds. \label{fig:fc_histogram}}
\end{figure}


\begin{figure}[hbt]
    \centering
    \begin{tabular}{ccc}
        \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.33\linewidth]{../../result/report_plots_pipeline/scatter_ID_triqler_vs_top3.png} %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.33\linewidth]{./img/scatter_ID_triqler_vs_top3.png} \DIFaddendFL & 
        \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.33\linewidth]{../../result/report_plots_pipeline/scatter_ID_triqler_vs_msstats.png} %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.33\linewidth]{./img/scatter_ID_triqler_vs_msstats.png} \DIFaddendFL & 
        \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.33\linewidth]{../../result/report_plots_pipeline/scatter_ID_triqler_vs_msqrob2.png} %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.33\linewidth]{./img/scatter_ID_triqler_vs_msqrob2.png} \DIFaddendFL \\
        A & B & C
    \end{tabular}
    \caption{{\bf Comparison of the summarized protein fold changes between the methods.} The comparison between the distributions of the MAP estimates from Triqler and (A) Top3, (B) MSstats, and (C) MSqRob2 for the data in the ID pipeline. The actual pipetted fold changes are indicated by dashed lines. Proteins not quantified by both compared methods were excluded from the plots. See Supplementary Figure \DIFdelbeginFL \DIFdelFL{\ref{fig:fc_scatter_supplement_triqler_vs_method_supplement} }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{\ref{fig:fc_scatter_triqler_vs_method_supplement} }\DIFaddendFL for protein-level results for both ID and PS workflows. \label{fig:fc_scatter}}
\end{figure}




\subsubsection*{Comparison of ability to discriminate differentially from equivalently abundant proteins}

As the first quantitative test of performance, we compared the methods' reported number of differentially abundant {\em E. Coli} and yeast proteins as a function of the number of HeLa proteins (Figure \ref{fig:diff_vs_hela}). As the former two lysates were injected in different concentrations and the HeLa was at constant concentration over the sample groups, a higher number of non-HeLa proteins for a similar number of HeLa-proteins is seen as better performance.  Overall, Triqler reports more \DIFdelbegin \DIFdel{differentially abundant protein for every non-differentially abundant protein }\DIFdelend \DIFaddbegin \DIFadd{such expected differentially abundant non-HeLa proteins per HeLa protein than the compared methods }\DIFaddend for both the peptide abundances generated by the ID spectral library and PS spectral library pipelines. The results can be found in Supplementary Figure \ref{fig:ability_to_differentiate_differentially_abundant_specie_vs_hela}. \DIFaddbegin \DIFadd{We also ran a second experiment where we removed shared peptides after matching, to ensure that the order of removal of the shared peptides does not affect performance. However, we found no large differences between removing peptides before or after database searching (Supplementary Figure \ref{fig:ability_to_differentiate_differentially_abundant_specie_vs_hela_full_lib}).
}\DIFaddend 

\begin{figure}[hbt]
    \centering
    \DIFdelbeginFL %DIFDELCMD < \begin{tabular}{lclc} 
%DIFDELCMD <         %%%
\DIFdelFL{A }\DIFdelendFL \DIFaddbeginFL \begin{tabular}{cc} 
        \includegraphics[width=0.45\linewidth]{./img/diff_HeLa_vs_nonHeLa_ID_all_0.51.png} \DIFaddendFL & 
        \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.45\linewidth]{../../result/report_plots_pipeline/diff_HeLa_vs_nonHeLa_ID_all_0.51.png} %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.45\linewidth]{./img/diff_HeLa_vs_nonHeLa_PS_all_0.51_Limma.png}\\
        \DIFaddFL{A }\DIFaddendFL & B 
    \DIFdelbeginFL %DIFDELCMD < & \includegraphics[width=0.45\linewidth]{../../result/report_plots_pipeline/diff_HeLa_vs_nonHeLa_PS_all_0.51.png} \\
%DIFDELCMD <     %%%
\DIFdelendFL \end{tabular} 
    \caption{{\bf Comparison of ability to differentiate proteins with \DIFdelbeginFL \DIFdelFL{differential }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{differentially }\DIFaddendFL abundance between conditions\DIFaddbeginFL \DIFaddFL{.}\DIFaddendFL } We plotted the number of reported differentially abundant  {\em E. Coli} and yeast proteins as a function of the number of proteins from the HeLa background when sorting according to significance for (A) ID pipeline and (B) PS pipeline. For the test, we selected a fold change evaluation of 0.51 for Triqler and a fold change threshold of 0.51 for Top3, MSstats and MSqRob2. \DIFdelbeginFL \DIFdelFL{All methods had a protein-level FDR threshold of 0.01. (}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{See the }\DIFaddendFL Supplement \DIFdelbeginFL \DIFdelFL{Figure }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{Figures }\DIFaddendFL \ref{fig:ability_to_differentiate_differentially_abundant_specie_vs_hela}\DIFdelbeginFL \DIFdelFL{shows differential abundance }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{-\ref{fig:ability_to_differentiate_differentially_abundant_specie_vs_hela_full_lib} for the number of differentially abundant proteins }\DIFaddendFL for each \DIFdelbeginFL \DIFdelFL{specie)}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{proteome}\DIFaddendFL . \label{fig:diff_vs_hela}}
\end{figure}

\subsubsection*{Comparison of statistical calibration}

Further, we tested the statistical calibration of the summarization methods. We hence investigated the relationship between the fraction of wrongly reported \DIFdelbegin \DIFdel{differential }\DIFdelend \DIFaddbegin \DIFadd{differentially }\DIFaddend abundant proteins (i.e. the fraction of HeLa proteins among all reported \DIFdelbegin \DIFdel{differential }\DIFdelend \DIFaddbegin \DIFadd{differentially }\DIFaddend abundant proteins), and each inference method's estimated false discovery rate (See Figure \ref{fig:frac_hela_vs_fdr} and Figure \ref{fig:frac_hela_vs_fdr_supp}). We observed that Triqler was better calibrated (closer to the diagonal line) than MSstats and MSqRob2. Top3 showed an even better calibration than Triqler but, as previously demonstrated, has a much lower sensitivity.

%DIF < We observed that Triqler, Top3, and MSqRob2 give estimates that are close to the true fold change level than MSstats. This is somewhat surprising given that both MSstats and MSqRob2 are using the Benjamini-Hochberger corrections which are generally found more conservative than $q$~value estimates.
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
%DIF < We observed that Triqler and Top3 were reporting slightly more accurate estimates than MSstats and MSqRobSum. This is somewhat surprising given that both MSstats and MSqRobSum are using the Benjamini-Hochberger corrections which are generally found more conservative than $q$~value estimates.
%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \begin{figure}[hbt]
    \centering
    \begin{tabular}{c} 
        \DIFdelbeginFL %DIFDELCMD < \includegraphics[width=0.5\linewidth]{../../result/report_plots_pipeline/calibration_ID_0.png}
%DIFDELCMD <     %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width=0.5\linewidth]{./img/calibration_ID_0.png}
    \DIFaddendFL \end{tabular}
  \caption{{\bf Comparison of calibration of the compared summarization methods.} We plotted the fraction of reported differentially abundant HeLa proteins as a function of the $q$~value threshold for protein abundances without any restrictions on the fold change. We used $q$~value for Triqler and Top3, and Benjamini-Hochberg corrections for MSstats and MSqRob2. \label{fig:frac_hela_vs_fdr}}
\end{figure}


%DIF < \subsubsection*{Differential abundance reported by the protein quantification tools.}
\DIFaddbegin \subsection*{\DIFadd{Benchmark on cancer proteomics-related data}}
\DIFaddend 

%DIF < We also investigated the number of reported differential abundant proteins as reported by our four protein summarization methods. The results can be found in Supplementary Figure \ref{fig:ability_to_differentiate_differentially_abundant_specie_vs_hela}. Triqler reports the most differentially abundant proteins per .
\DIFaddbegin \DIFadd{To further compare the different protein inference methods, they were finally tested on two sets of less engineered data --- sets without known relative abundance differences.  Data was downloaded from Stewart }{\em \DIFadd{et al.}}\DIFadd{, a set consisting of DIA data contrasting lung squamous cell carcinomas to adjacent tissues~}\cite{stewart2017relative}\DIFadd{, and from Gao }{\em \DIFadd{et al.}} \DIFadd{a set comparing primary to metastatic cell lines}\cite{gao2021data}\DIFadd{. Peptide abundances were derived with the nf-core quantms pipeline followed by our protein summarization strategies and plotted the number of differentially abundant proteins as a function of each method's estimated quantitative protein-level FDR of $q$~value. We used fold change evaluation of 0.92 resp. 0.63 for Triqler, based on its lower bound estimate. We used the the same lower bound estimate as a fold change threshold when reporting performance of the compared methods. As can be seen in Figure~\ref{fig:non-engineered}, Triqler reports more differential proteins than the other methods for similair estimated $q$~values.
}


\begin{figure}[hbt]
    \centering
    \begin{tabular}{cc} 
        \includegraphics[width=0.45\linewidth]{./img/pxd004684.png} & 
        \includegraphics[width=0.45\linewidth]{./img/pxd022992.png} \\
        \DIFaddFL{A }& \DIFaddFL{B }\\
    \end{tabular}
  \caption{{\bf \DIFaddFL{Reported performance of the compared methods on two cancer proteomics-related datasets.}} \DIFaddFL{We plotted the reported differentially abundant proteins as a function of their reported FDR/$q$~value for the (A) Stewart }{\em \DIFaddFL{et al.}} \DIFaddFL{set (lung squamous cell carcinomas vs. adjacent tissues) and (B) Gao }{\em \DIFaddFL{et al.}} \DIFaddFL{(primary vs. metastatic cell lines). }\label{fig:non-engineered}}
\end{figure}
\DIFaddend 


\section*{Discussion}

Here we have shown that Triqler operates well for DIA data, despite \DIFaddbegin \DIFadd{being }\DIFaddend originally intended for DDA data. We also find that Triqler outperforms other protein summarization methods on an engineered benchmark set, both in terms of sensitivity and accuracy in its error estimates. Triqler was also able to detect a higher number of differentially abundant proteins at a more accurately reported false discovery rate than the compared methods. The absence of filtering and imputation steps before Triqler benefits the analysis by making it both more user-friendly by reducing parameter choices and inducing less bias into the result. 

The analytes in shotgun proteomics are peptides and not proteins or proteoforms. Nevertheless, most users of mass spectrometry use and will continue to find reasons to report findings on a protein level. It makes sense to put efforts into a better understanding of which protein inference tools to use at what occasion and how to summarize peptide abundances into protein relative concentration values. Also, protein summarization gives lower variance than peptide-level analysis, and it reduces the number of hypotheses tested and reduced the number of missing values, which can have a major impact on the quality of the analysis \cite{plubell2021can}.   

One important remark is that the sequence database that we used for matching the spectra was filtered so that only one protein per peptide was kept, to control for any difference in protein inference strategies used by our compared protein summarization methods. There is currently no consensus on how to handle multiple proteoforms in bottom-up proteomics. Hence, we believe that protein inference strategies that can account for multiple proteoforms would greatly benefit the field by improving the quality of the quantitative analysis. 

We also see some differences in how DIA and DDA peptide-level abundance data appear. For instance, there are more missing values in the DDA than in the DIA data. We addressed this issue by providing an alternative method for estimating the censoring function for missing values in Triqler.

It is quite hard to evaluate how the performance of a data processing pipeline is influenced by its components. This should not stop the field from trying to establish the features of the different processing steps \cite{dufresne2014abrf,gatto2016testing,navarro2016multicenter}. Unbiased comparisons of software tools are challenging for several reasons \cite{dufresne2014abrf}. Methods can be assessed by scientists lacking relevant expertise, the tested methods may be lacking sufficient documentation and the interpretation of test results may be subjective \cite{yates2012toward,leprevost2014best,pak2013clustering,faircomparison2015}. By using the same data set we can assure that the data set is processed consistently and further the analysis by extending it to protein summarization procedures.

Lastly, we want to highlight the benefit and importance of datasets such as the one provided by Navarro et al. \cite{navarro2016multicenter}. These benchmarking datasets make it easy for the scientific community to investigate computational tools by providing a golden standard and significantly facilitating benchmark studies. 

\DIFaddbegin \section*{\DIFadd{Supporting Information}}
\DIFadd{The following supporting information is available free of charge at ACS website }\url{http://pubs.acs.org}
\begin{itemize}
\item \DIFadd{Table S1: Protein count in the Uniprot FASTA protein database.
}\item \DIFadd{Table S2: Number of identified peptides and proteins for the ID workflow.
}\item \DIFadd{Table S3: Number of identified peptides and proteins for the PS workflow
}\item \DIFadd{Figure S1: Standard deviation of peptide abundance as a function of mean abundance in
DIA experiments.
}\item \DIFadd{Figure S2: Comparison of actual missing values against fit to the censoring distribution
used in Triqler. 
}\item \DIFadd{Figure S3: Comparison of reported log2-fold change between Triqler and the compared
methods.
}\item \DIFadd{Figure S4: The compared methods’ ability to distinguish differentially abundant proteins
when applying protein-level fold-change tresholds.
}\item \DIFadd{Figure S5: The compared methods’ ability to distinguish differentially abundant proteins
when not applying fold change treshholds.
}\item \DIFadd{Figure S6: The compared methods’ ability to distinguish differentially abundant proteins
when removing shared peptides after database matching.
}\item \DIFadd{Figure S7: Comparison of reported fold change distributions.
}\item \DIFadd{Figure S8: Comparison of calibration of the compared summarization methods.
}\end{itemize}

\DIFaddend \section*{Funding}

This work was supported by grants from the Swedish Research Council (grant 2017-04030).

\section*{\DIFdelbegin \DIFdel{Supporting information}\DIFdelend \DIFaddbegin \DIFadd{Acknowledgements}\DIFaddend }
\DIFaddbegin 

\DIFadd{We would like to thank Yasset Perez-Riverol, EMBL-EBI for helpful discussions.
}\DIFaddend 


\bibliographystyle{unsrt}
%\bibliography{benchmark}
\bibliography{benchmark.bib}
\end{document}



