\documentclass[11pt]{article}
\usepackage[margin=2cm,a4paper]{geometry}
\usepackage{graphicx}
\usepackage{authblk}
\usepackage{url}
\usepackage{array}
\usepackage{amsfonts}
\usepackage{multirow}
\usepackage{amsmath}
\usepackage[usenames, dvipsnames]{color, colortbl}
\usepackage[normalem]{ulem}
\usepackage{todonotes}
\usepackage[outdir=./img/]{epstopdf}
\usepackage{epsfig}

\usepackage{tikz}
\usetikzlibrary{fit,positioning}
\usetikzlibrary{arrows}

\makeatletter
\def\title#1{\gdef\@title{Supplement to: ``#1''}}
\makeatother
\renewcommand{\thetable}{S\arabic{table}}%
\renewcommand{\thefigure}{S\arabic{figure}}%
\renewcommand{\thepage}{S\arabic{page}}

\setlength\parindent{0pt}


% supplement referencing
\usepackage{subcaption}
%\usepackage{cleveref}

% tablet style table
 \usepackage{booktabs}
 
\input{./title.tex}

\begin{document}

\maketitle

\section*{Note S1: Supplementary figures and tables}
\label{sec:fc-eval}

\section*{Supplementary Tables}

\begin{table}[h]
    \begin{tabular}{lllllll}
    \hline
    \multicolumn{7}{c}{ID workflow}                                                                                                                                                                                   \\ \hline
    Condition & \multicolumn{3}{c}{1}                                                                         & \multicolumn{3}{c}{2}                                                                         \\
    Filename       & \multicolumn{1}{c}{002-Pedro} & \multicolumn{1}{c}{004-Pedro} & \multicolumn{1}{c}{006-Pedro} & \multicolumn{1}{c}{003-Pedro} & \multicolumn{1}{c}{005-Pedro} & \multicolumn{1}{c}{007-Pedro} \\
    Peptides  & 12 934                        & 13 819                        & 13 063                        & 12 023                        & 14 858                        & 15 208                        \\
    Proteins  & 2 252                         & 2 321                         & 2 243                         & 2 159                         & 2 427                         & 2 433                         \\ \hline
    \end{tabular}
     \caption{{\bf Number of identified peptides and proteins} for the samples using the ID workflow and a peptide-level FDR at 0.01. The FDR-level is obtained with an $m\_score$ cutoff at 0.00079. This $m\_score$ cutoff is computed by setting the desired peptide-level FDR of 0.01 with \texttt{mscore4pepfdr} in the \texttt{SWATH2stats} package.
          \label{fig:osw_peptide_and_protein_id}}
\end{table}
        

\begin{table}[h]
    \begin{tabular}{lllllll}
    \hline
    \multicolumn{7}{c}{PS workflow}                                                                                                                                                                                   \\ \hline
    Condition & \multicolumn{3}{c}{1}                                                                         & \multicolumn{3}{c}{2}                                                                         \\
    Filename       & \multicolumn{1}{c}{002-Pedro} & \multicolumn{1}{c}{004-Pedro} & \multicolumn{1}{c}{006-Pedro} & \multicolumn{1}{c}{003-Pedro} & \multicolumn{1}{c}{005-Pedro} & \multicolumn{1}{c}{007-Pedro} \\
    Peptides  & 20 880                        & 20 653                        & 20 907                        & 21 118                        & 21 192                        & 21 137                        \\
    Proteins  & 3 228                         & 3 224                         & 3 240                         & 3 243                         & 3 255                         & 3 249                         \\ \hline
    \end{tabular}
     \caption{{\bf Number of identified peptides and proteins} for the samples using the PS workflow with a peptide-level FDR at 0.01.
          \label{fig:diann_peptide_and_protein_id}}
\end{table}




\subsubsection*{Comparison of ability to differentiate differentially abundant proteins}
\begin{figure}[hbt]
    \centering
    \begin{tabular}{lclc} 
        A & \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/diff_HeLa_vs_nonHeLa_ID_ecoli_0.51.png} & 
        D & \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/diff_HeLa_vs_nonHeLa_PS_ecoli_0.51.png} \\ 
        B & \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/diff_HeLa_vs_nonHeLa_ID_yeast_0.51.png} & 
        E & \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/diff_HeLa_vs_nonHeLa_PS_yeast_0.51.png} \\
        C & \includegraphics[width=0.45\linewidth]{../../result/report_plots_pipeline/diff_HeLa_vs_nonHeLa_ID_all_0.51.png} & 
        F & \includegraphics[width=0.45\linewidth]{../../result/report_plots_pipeline/diff_HeLa_vs_nonHeLa_PS_yeast_0.51.png} \\ 

    \end{tabular}
    \caption{{\bf The compared methods ability to differentiate differentially abundant proteins.} We plotted the number of reported differentially abundant  {\em E. Coli} and Yeast proteins as a function of number of proteins from the HeLa background when sorting according to significance for (A) SL pipeline and (B) PS pipeline. For the test we selected a fold-change evaluation of 0.51 for Triqler and fold-change treshold of 0.51 for Top3, MSstats and MSqRob2. All methods have an protein-level FDR threshold of 0.01. \label{fig:ability_to_differentiate_differentially_abundant_specie_vs_hela}}
\end{figure}


\subsubsection*{Investigation of the standard deviations as a function of abundance in DIA experiments.}
\begin{figure}[hbt]
    \centering
    \centering
    \begin{tabular}{lclc} 
        A & \includegraphics[width=0.5\linewidth]{../../result/report_plots_pipeline/quantile_bins_ID_median.png} &
        B & \includegraphics[width=0.5\linewidth]{../../result/report_plots_pipeline/quantile_bins_PS_median.png} \\
    \end{tabular}
  \caption{{\bf Standard deviation of peptide abundance as a function of mean abundance in DIA experiments.} We performed quantile binning using pandas qcut function and plotted the standard deviation across samples as a function of the mean of every peptide intensity in the TripleTOF6600 section of the LFQ Bench set. The x-axis value indicate the median of the bin-range. We used a log-log scale for abundances derived by the (A) ID pipeline and (B) PS pipeline.  We observe a nearly uniform offset in standard deviation across the intensity scale, demonstrating that $\log(\sigma) \approx \log(\mu) + \log(k)$ and hence   $\sigma \approx \mu k$. Linear scale plots are reported as reference.  \label{fig:uniform_offset_in_standard_deviation_boxplot}}
\end{figure}


\begin{table}[hbt]
\centering
\begin{tabular}{llll}
\hline
        & Unfiltered & no\_shared & no\_shared\_IL\_equivalence \\ \hline
All     & 31 055     & 30 456     & 30 452                      \\
E. Coli & 4 391      & 4 306      & 4 306                       \\
Human   & 20 614     & 20 302     & 20 299                      \\
Yeast   & 6 050      & 5 848      & 5 847                       \\ \hline
\end{tabular}

  \caption{{\bf Protein count in the Uniprot FASTA protein database.} The database is a FASTA file with one protein sequence per gene for each species (UP000005640, UP000000625 and UP000002311. Acquired on 2021-06-16). The "no\_shared" filter is applied by splitting the protein sequences at amino acids "K" and "R" and keeping all the sequences with length $>$ 7 for each protein. We then mapped each sequence to all possible protein matchings and counted the how many proteins each split sequence was mapped to, and filtered so that each sequence only kept one protein match. Therefore, creating a database library with only one peptide sequence per protein. For the "no\_shared\_IL\_equivalence" all I are replaced by L before performing the filtering. \label{table:proteins_in_database}}

\end{table}



\subsubsection*{Comparison of statistical calibration}
\begin{figure}[hbt]
    \centering
    \centering
    \begin{tabular}{lclc} 
        A \includegraphics[width=0.5\linewidth]{../../result/report_plots_pipeline/fraction_missing_values_ID.png} & &%\includegraphics[width=0.3\linewidth]{} & 
        B \includegraphics[width=0.5\linewidth]{../../result/report_plots_pipeline/fraction_missing_values_PS.png} & \\%\includegraphics[width=0.3\linewidth]{} \\ 
    \end{tabular}
    \caption{{\bf Comparison of actual missing values against fit to the censored normal distribution used in triqler.} We imputed the missing values as the mean of sample peptide intensities and used these imputed values to approximate the missingness for a given intensity. We binned the intensities to an arbitrary small range and plotted the fraction os missing values for each intensity range. A cubic spline fit was used to fit the values against the mentioned censored normal distribution for (A) SL pipeline  and (B) PS pipeline. \label{fig:fraction_missing_values}}

\end{figure}


\subsubsection*{Comparison of reported protein count}
\begin{figure}[hbt]
    \centering
    \newcolumntype{V}{>{\centering\arraybackslash} m{.4\linewidth} }
    \setlength{\tabcolsep}{0pt}
    %\includegraphics[width=16cm]{../../result/2021-08-13_docs_plots/intensity_plot.png}
    
    %\begin{tabular}{>{\centering\arraybackslash} m{3cm} V V} 
    %    & ID & PS \\
    %    Triqler & \includegraphics[trim=0 70 0 -70, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_ID_triqler.png} & 
    %    \includegraphics[trim=0 70 0 -70, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_PS_triqler.png} \\ 
    %    MSqRob2 &\includegraphics[trim=0 70 0 -70, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_ID_msqrob2.png} & 
    %    \includegraphics[trim=0 70 0 -70, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_PS_msqrob2.png} \\ 
    %    MSstats & \includegraphics[trim=0 70 0 -70, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_ID_msstats.png} & 
    %    \includegraphics[trim=0 70 0 -70, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_PS_msstats.png} \\ 
    %    Top3 & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/histogram_ID_top3.png} &
    %    \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/histogram_PS_top3.png} 
    %\end{tabular}
    
    \begin{tabular}{>{\centering\arraybackslash} m{3cm} V V} 
        & ID & PS \\
        Triqler & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/histogram_ID_triqler.png} & 
        \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/histogram_PS_triqler.png} \\ 
        MSqRob2 & \vspace{-1cm}\includegraphics[trim={0 0 0 6}, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_ID_msqrob2.png} & 
        \vspace{-1cm}\includegraphics[trim={0 0 0 6}, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_PS_msqrob2.png} \\ 
        MSstats & \vspace{-1cm}\includegraphics[trim={0 0 0 16}, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_ID_msstats.png} & 
        \vspace{-1cm}\includegraphics[trim={0 0 0 18}, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_PS_msstats.png} \\ 
        Top3 & \vspace{-0.9cm}\includegraphics[trim={0 0 0 10}, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_ID_top3.png} &
        \vspace{-1.1cm}\includegraphics[trim={0 0 0 15}, clip, width=\linewidth]{../../result/report_plots_pipeline/histogram_PS_top3.png} 
    \end{tabular}

    \caption{{\bf Comparison of reported fold change distributions.} We used protein data from (A-D) ID and (E-H) PS pipelines as generated by 
    (A,E) Triqler, (B,F) MSqRob2, (C,G) MSstats, and (D,H) Top3. \label{fig:fc_histogram_supplement}}
\end{figure}

\subsubsection*{Summarized protein abundances}
\begin{figure}[hbt]
    \centering
    \newcolumntype{V}{>{\centering\arraybackslash} m{.4\linewidth} }
    \setlength{\tabcolsep}{0pt}
   %\includegraphics[width=16cm]{../../result/2021-08-13_docs_plots/intensity_plot.png}
    \begin{tabular}{   >{\centering\arraybackslash} m{3cm} V V}
                & ID & PS \\
        {\rotatebox[origin=c]{90}{Triqler}} & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/scatter_ID_triqler.png}  
                & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/scatter_PS_triqler.png} \\ 
        {\rotatebox[origin=c]{90}{MSqRob2}} & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/scatter_ID_msqrob2.png}  
                & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/scatter_PS_msqrob2.png} \\ 
        {\rotatebox[origin=c]{90}{MSstats}} & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/scatter_ID_msstats.png}  
                & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/scatter_PS_msstats.png} \\ 
        {\rotatebox[origin=c]{90}{Top3}}    & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/scatter_ID_top3.png}
                & \includegraphics[width=\linewidth]{../../result/report_plots_pipeline/scatter_PS_top3.png} 
    \end{tabular}
    \caption{{\bf Summarized protein abundances by the compared methods.} from (A-D) ID and (E-H) PS pipelines as generated by 
    (A,E) Triqler, (B,F) MSqRob2, (C,G) MSstats, and (D,H) Top3. \label{fig:fc_scatter_supplement}}
\end{figure}



\subsubsection*{Comparison of statistical calibration}
\begin{figure}[hbt]
    \centering
    \centering
    \begin{tabular}{lclc} 
        A \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/calibration_ID_0.png} & &%\includegraphics[width=0.3\linewidth]{} & 
        C \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/calibration_ID_0.51.png} & \\%\includegraphics[width=0.3\linewidth]{} \\ 
        B \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/calibration_PS_0.png} & &%\includegraphics[width=0.3\linewidth]{} & 
        D \includegraphics[width=0.4\linewidth]{../../result/report_plots_pipeline/calibration_PS_0.51.png} & \\%\includegraphics[width=0.3\linewidth]{} \\ 
    \end{tabular}
  \caption{{\bf Comparison of calibration of the compared summarization methods.} We plotted the fraction of reported differentially abundant HeLa proteins as a function of $q$~value for (A,C) ID pipeline and (B,D) PS pipeline. (A,C) is non FC-thresholded and (C,D) is FC-thresholded at 0.51. \label{fig:frac_hela_vs_fdr}}
\end{figure}


\subsubsection*{Number of peptides per protein for Triqler estimated fold change regions}
\begin{figure}[hbt]
    \centering
    \centering
    \begin{tabular}{lclc} 
        A \includegraphics[width=0.5\linewidth]{../../result/report_plots_pipeline/fc_peptide_count_ID_triqler.png} & &
        B \includegraphics[width=0.5\linewidth]{../../result/report_plots_pipeline/fc_peptide_count_PS_triqler.png} & \\
    \end{tabular}
    \caption{{\bf Proteins with a different Triqler-estimated abundance than expected are summarized by less peptides than ones that are well estimated.} We plotted the median number of peptides per protein for    
  We binned the peptides into regions of 0.1 ranged bins of estimated fold change and plotted them median for all bins between [-2,3], and plotted the median number of peptides as a function of bin-centers of the fold change. (A) is the ID pipeline and (B) is the PS pipeline \label{fig:number_of_peptides_supplement}}
\end{figure}



\end{document}
